src/components/__tests__/Workspace.test.js

```diff
@@ -368,7 +371,7 @@ describe('The Workspace component', () => {
     workspace.update();
     workspace.instance().runCode();
 
-    expect(setTimeout).toHaveBeenCalled();
+    expect(setTimeout).toHaveBeenCalledTimes(2);
```

Comment:
The debouncer is calling `setTimeout` internall?

Reply:
Yes